cinema/views.py

```diff
@@ -81,12 +102,15 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         MovieSession.objects.all()
         .select_related("movie", "cinema_hall")
         .annotate(
-            tickets_available=F("cinema_hall__rows")
-            * F("cinema_hall__seats_in_row")
-            - Count("tickets")
+            tickets_available=F(
+                "cinema_hall__rows") * F(
+                "cinema_hall__seats_in_row") - Count(
+                "tickets")
         )
```

Comment:
You have bad formatting here
Parentheses should start and end with a text or have a newline before a text both at the start and the end